img/module-graph-cycle-async.svg

```diff
@@ -0,0 +1,69 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 241 211">
+<style>
+  rect {
+    fill: white;
+    stroke: black;
+    rx: 10;
+    ry: 10;
+  }
+
+  text {
+    dominant-baseline: central;
+    text-anchor: middle;
+    font-family: sans-serif;
+  }
+
+  line {
+    stroke: black;
+    marker-end: url(#arrowhead);
+  }
+</style>
+
+<defs>
+  <marker id="arrowhead" markerWidth="10" markerHeight="10" refX="10" refY="3" orient="auto">
+  <path d="M0,0 L0,6 L9,3 z"></path>
+  </marker>
+</defs>
+
+<rect x="60.5" y="0.5" width="60" height="30"/>
+<text x="90.5" y="15.5">A.js</text>
```

Comment:
The other graphics don't use any suffixes for the module names, just A, B, etc. Should do the same here and below.

Reply:
if there were an extension, it should be `.mjs`, which has an existing example in the spec and matches node, the primary JS implementation that has the concept of file extensions - but removing the extension seems better.